scripts/shared/db-setup.js

```diff
@@ -145,7 +179,7 @@ export async function setupDatabase(options) {
 
 			if (!forceDropConfirm) {
 				console.log(chalk.red('\n⚠️  WARNING: This will DELETE ALL DATA in the following tables:'))
-				console.log(chalk.red('  - bundles, cids, balances, nonces, proposers, vaults'))
+				console.log(chalk.red('  - bundles, cids, balances, nonces, proposers'))
```

Comment:
This line should keep `vaults`, but also add `deposit_assignment_events` and `deposits`.